lib/virt_autotest/utils.pm

```diff
@@ -151,22 +187,30 @@ sub ensure_online {
 
     my $hypervisor = $args{HYPERVISOR}    // "192.168.122.1";
     my $dns_host   = $args{DNS_TEST_HOST} // "suse.de";
+    my $skip_ssh   = $args{skip_ssh}      // 0;
     # Ensure guest is running
     # Only xen/kvm support to reboot guest at the moment
     if (is_xen_host || is_kvm_host) {
         if (script_run("virsh list --all | grep '$guest' | grep running") != 0) {
             assert_script_run("virsh start '$guest'");
         }
     }
-    die "$guest does not respond to ICMP" if (script_retry("ping -c 1 '$guest'", delay => 5, retry => 60) != 0);
+    die "$guest does not respond to ICMP" if (script_retry("ping -c 1 '$guest'", delay => 45, retry => 60) != 0);
```

Comment:
`60*45` - do we really want to try to ping a guest for 45 minutes?

Reply:
This should be probably variable then. We do this directly after the installation and the installation may take ~30 minutes on busy hypervisor.

Reply:
Did:
```perl
my $ping_delay = $args{ping_delay}    // 15;
my $ping_rerty = $args{ping_retry}    // 60;

# …

die "$guest does not respond to ICMP" if (script_retry("ping -c 1 '$guest'", delay => $ping_delay, retry => $ping_retry) != 0);
```
And in `tests/virtualization/universal/waitfor_guests.pm` I did:
```perl
ensure_online $_, skip_ssh => 1, ping_delay => 45 foreach (keys %virt_autotest::common::guests);
```